Mathlib/Algebra/Homology/HomotopyCategory/ShiftSequence.lean

```diff
@@ -119,8 +119,8 @@ instance {K L : CochainComplex C ℤ} (φ : K ⟶ L) (n : ℤ) [QuasiIso φ] :
     QuasiIso (φ⟦n⟧') where
   quasiIsoAt a := by
     rw [quasiIsoAt_iff_isIso_homologyMap]
-    refine' (NatIso.isIso_map_iff
-      ((homologyFunctor C (ComplexShape.up ℤ) 0).shiftIso n a (n + a) rfl) φ).2 _
+    refine (NatIso.isIso_map_iff
```

Comment:
I feel like this should be `apply` (or perhaps we should discuss deprecating `apply` in favor of `refine`).